### DISCUSSION TITLE
token refresh problem that in nestjs and prisma

### DISCUSSION THREAD

**Discussion by StephenWuuu** [prisma/prisma#17362] [Q&A]
### Bug description

i use nestjs and prisma to pratice how to make a jwt authoriazation , but when i use the nestjs update controller, the problem came up

```Invalid this.prisma.user.update() invocation in
C:\Users\SC\Desktop\nest-prisma\src\users\users.service.ts:25:29
  22 }
  23
  24 async update(id: string, updateUserDto: UpdateUserDto) {
→ 25   return this.prisma.user.update({
         data: {
           refreshToken: '$argon2id$v=19$m=65536,t=3,p=4$2vVDupRSjM9KCgHgEh7xxQ$lquXuVnH8aDnFEpIn+H8eGZf/Qpvn2Phugq4Q4M9dxs'
           ~~~~~~~~~~~~
         },
         where: {
           id: 'clcwjo2vu0000vm9gpldg0kjv'
         }
       })
```

Unknown arg `refreshToken` in data.refreshToken for type UserUpdateInput. Available args:
```graphql
type UserUpdateInput {
  id?: String | StringFieldUpdateOperationsInput
  email?: String | StringFieldUpdateOperationsInput
  password?: String | StringFieldUpdateOperationsInput
  name?: String | NullableStringFieldUpdateOperationsInput | Null
  role?: Role | EnumRoleFieldUpdateOperationsInput
  profile?: ProfileUpdateOneWithoutUserNestedInput
  article?: ArticleUpdateManyWithoutAuthorNestedInput
  createdAt?: DateTime | DateTimeFieldUpdateOperationsInput
  updatedAt?: DateTime | DateTimeFieldUpdateOperationsInput
}
``` 

it seems that i have get the token ,but i don't know about the problem how it happend??

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by mcbethio** [comment]
Looking at your example here, `refreshToken` doesn't exist on UserUpdateInput.

But, I think we need to know more about how you are using Nest & Prisma first. Would you be able to provide us with a reproduction repo and that would really help us support you.

**Reply by ludralph** [reply]
Hi @StephenWuuu 

It would be really helpful if you could provide the additional information that we had asked about last time. If your problem is resolved by now or somehow doesn't feel relevant any more, you can simply close the Discussion as well.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
